modules/sugar/src/test/ts/browser/CssTransferTest.ts

```diff
@@ -22,21 +24,21 @@ UnitTest.test('CssTransfer', function () {
     return r;
   };
 
-  const gamma = function () {
+  const gamma = () => {
     const r = Div();
     Css.setAll(r, {
       'background-color': 'red'
     });
     return r;
   };
 
-  const check = function (expectedPresent, expectedAbsent, source, destination, styles) {
+  const check = (expectedPresent: Record<string, string>, expectedAbsent: string[], source: Element<DomElement>, destination: Element<DomElement>, styles: string[]) => {
     Css.transfer(source, destination, styles);
-    Arr.each(expectedAbsent, function (k) {
+    Arr.each(expectedAbsent, (k) => {
       if (Css.getRaw(destination, k).isSome()) { assert.fail('Result should not have style: ' + k); }
```

Comment:
Surprised you didn't reformat this to put it on two lines?

Reply:
I wasn't aiming to fix formatting issues, only replace functions with arrow functions.

Reply:
Fixed